server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -187,7 +192,18 @@ function markAppointmentAsReady($token, $firstName, $lastName, $emailAddress, $p
 				<b>Preferred Language:</b> $preferredLanguage <br/>
 				<b>Email (if provided):</b> $emailAddress <br/>
 				<b>Type:</b> $appointmentType";
-			EmailUtilities::sendHtmlFormattedEmail($toEmailString, 'VITA -- Appointment Ready', $readyMessage);
+			EmailUtilities::sendHtmlFormattedEmail($toEmailString, 'VITA -- Appointment Ready', $volunteerMessage);
+
+			if(!empty($emailAddress)) {
+				// Email client confirmation
+				$clientMessage = "Congratulations, $firstName $lastName, you have successfully marked your appointment as ready!<br/>
+					<b>Site:</b> $siteName".(AppointmentTypeUtilities::isVirtualAppointmentType($appointmentType) ? " (Please do not show up to the site for your virtual appointment)" : "")."<br/>
+					<b>Your Phone Number:</b> $phoneNumber <br/>
+					<b>Your Chosen Best Time to Call:</b> $bestTimeToCall <br/> 
+					<b>Your Preferred Language:</b> $preferredLanguage <br/>
+					<b>Appointment Type:</b> $appointmentType";
```

Comment:
You should probably remove this from the client email, I don't see a compelling reason to tell them this information

Reply:
Do you mean just appointment type? I figured it was one more chance to let them know their appointment wasn't a real face-to-face thing

Reply:
Yah that's all I thought should be deleted.. Ig you give a good point. What were the values that would be displayed for $appointmentType? I thought it was the "lookupType" name which isn't exactly user-friendly, that's what we use for programmatic convenience... If it's the actual name (i.e. the nice-looking one), then I think we're fine including it here

Reply:
No, not very user-friendly, I took it out